pymatgen/analysis/defects/point_defects.py

```diff
@@ -1530,8 +1530,9 @@ class StructureMotifInterstitial(Defect):
     __supported_types = ("tet", "oct", "bcc", "tetoct")
 
     def __init__(self, structure, inter_elem,
-                 motif_types=["tet", "oct", "tetoct"],
-                 op_targets=[1.0, 1.0, 1.0], op_threshs=[0.5, 0.5, 0.5],
+                 motif_types=None,
```

Comment:
Yes, use tuples for the ones which are specific. That way, people do not need to look into the code to know the default values. I do not know what the online fora say. They are wrong. The Hulk is right.